.bazelrc

```diff
@@ -35,6 +35,10 @@ build --nolegacy_external_runfiles
 # Don't bother building targets which aren't dependencies of the tests.
 test --build_tests_only
 
+# Default changed in bazel 8
+# See: https://github.com/bazelbuild/bazel/commit/489d08bf472f4568b3f7f478a8747a30cb956111
+build --zip_undeclared_test_outputs=true
```

Comment:
This is the easy solution to make it work with bazel 8. Getting rid of this flag might require some changes in the rostest impl of this repo.

Reply:
Interesting, thanks for digging into this. Do you know what changes are needed?